java-generator/core/pom.xml

```diff
@@ -99,8 +99,6 @@
     <dependency>
       <groupId>com.approvaltests</groupId>
       <artifactId>approvaltests</artifactId>
-      <version>13.3.0</version>
-      <scope>test</scope>
```

Comment:
this way we are adding `approvaltests` to the transitive dependencies, is it intended?

Reply:
👍